include/oneapi/dpl/pstl/glue_algorithm_impl.h

```diff
@@ -350,8 +350,7 @@ transform(_ExecutionPolicy&& __exec, _ForwardIterator1 __first1, _ForwardIterato
 {
```

Comment:
There is a comment on line 344:
`// we can't use non-const __op here` 

which should be removed with this PR, as it is no longer matches the implementation.

Reply:
Removed